.github/workflows/collect-log/action.yaml

```diff
@@ -0,0 +1,20 @@
+name: Log Collector
```

Comment:
Let's rename this with a plural name: `.github/workflows/collect-logs/action.yaml`

Let's also make a comment that it ISN'T to be used for the go integration tests because their logs are placed in a different location.

Reply:
added.